### ISSUE TITLE
Default params for Hello World. undefined vs empty?

### ISSUE THREAD

**Issue by matthewmorgan** [exercism/xecmascript#112]
So, I proposed to a team member that he should try using default params in hello-world.  However, it doesn't work because the first test passes an empty string rather than an undefined value.  Here's his solution: http://exercism.io/submissions/35fced9768e840df98afa4b85fbfac56

Should the first test be changed to pass an undefined value to allow the use of default params?  EG
current:
```
  it('says hello world with no name', () => {
    expect(helloWorld.hello('')).toEqual('Hello, World!');
  });
```
maybe?:
```
  it('says hello world with no name', () => {
    expect(helloWorld.hello()).toEqual('Hello, World!');
  });
```
@kytrinyx @rchavarria what do you think? What do the other tracks do with empty but not undefined values?

**Comment by kytrinyx** [comment]
Ruby doesn't test for an empty string, it only does no parameter (defaults to "World") and a specific parameter: https://github.com/exercism/xruby/blob/master/hello-world/hello_world_test.rb#L18-L31

I think the proposed change makes more sense.

**Comment by matthewmorgan** [comment]
OK submitted as PR #113 Thanks!

**Comment by rchavarria** [comment]
JavaScript track passes an empty string. But your proposal is better, because it allows the user to use a ES2015 functionality (the goal of this language track).

**Comment by rchavarria** [comment]
Closed by #113
